okta/resource_okta_policy_password_test.go

```diff
@@ -50,6 +50,7 @@ func TestAccOktaPolicyPassword_crud(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "name", buildResourceName(ri)),
 					resource.TestCheckResourceAttr(resourceName, "status", "ACTIVE"),
 					resource.TestCheckResourceAttr(resourceName, "description", "Terraform Acceptance Test Password Policy"),
+					resource.TestCheckResourceAttr(resourceName, "send_lockout_email.#", "[\"EMAIL\"]"),
```

Comment:
Looks like this assertion needs to be updated. Note the hash is the length of the set

```
resource.TestCheckResourceAttr(resourceName, "lockout_notifications.#", "1")
```